### DISCUSSION TITLE
Trying to get follower & following list, banned for spamming @ mentions.

### DISCUSSION THREAD

**Discussion by ArsenicBismuth** [tweepy/tweepy#1830] [Questions] [answered]
So yesterday, I was following this project to get Twitter follower list and make an infographic of user network. Project: https://towardsdatascience.com/how-to-download-and-visualize-your-twitter-network-f009dbbf107b

But after short sessions, my Twitter API access is limited. When I replied to it, I got this email just today stating that I was abusing @ mentions.
![image](https://user-images.githubusercontent.com/19264532/155890901-8e3426f2-31e0-497a-a665-a1771acf51b9.png)

Could anyone clarify what is being used here to warrant that ban? The only functionality I use out of Tweepy are: Getting user ID & follower list. How does this library somehow managed to spam mention someone?

**Comment by ArsenicBismuth** [comment, marked as answer]
Got a quick reply stating that it was a false alarm from their bulk-ban system. The whole resolution process resolved very quick, so if you got in trouble despite you're behaving as a good kid, don't worry to contact or reply to their inquiries.

**Comment by Harmon758** [comment]
In the future, any Twitter API policy questions are better directed towards [Twitter API Policy Support](https://help.twitter.com/en/forms/platform) or the [Twitter Developers Forums](https://twittercommunity.com/) instead.
